test/SvgGenerationTest.hx

```diff
@@ -269,6 +283,75 @@ class SvgGenerationTest
         }
 	}
     
+    // Compares 2 generated PNGs
+    private function compareGeneratedPNGs(pngName1:String, pngName2:String)
```

Comment:
I'm not really crazy about the amount of copy/paste here. If we decide to change our pixel-diff-calculation later, or find a bug in this code, we have to remember to fix it in two places.

Could you please refactor out the common code instead? Or, if that makes the code too complex or hard to understand, what else can we do here?